index.bs

```diff
@@ -475,9 +475,9 @@ the given [=sensor type=]. See [[FEATURE-POLICY]] for more details.
 
 <h4 id="focused-area" oldids="losing-focus">Focused Area</h4>
 
-[=Sensor readings=] are only available for [=active documents=] whose
-origin is [=same origin-domain=] with the [=currently focused area=]
-document.
+[=Sensor readings=] are only available for [=active documents=] whose origin is
+[=same origin-domain=] with the document of the [=currently focused area=] of
+the [=top-level browsing context=] of the specified [=active document=].
```

Comment:
I'm not sure which document the "of the specified active document" clause refers to.

Reply:
I *think* that the idea is that, to determine whether sensor readings are available within a particular document, you look at that document's  top-level browsing context, find the focused document inside it, and check whether the two documents are same-origin-domain. ... How about:

```suggestion
the same [=top-level browsing context=].
```

?